server/queries.js

```diff
@@ -54,18 +54,73 @@ const updateApplication = (request, response) => {
 const deleteApplication = (request, response) => {
     const id = parseInt(request.params.id)
 
-    pool.query('DELETE FROM applications WHERE id = $1', [id], (error, results) => {
+    pool.query('DELETE FROM applications WHERE application_id = $1', [id], (error, results) => {
         if (error) {
             throw error
         }
         response.status(200).send(`Application deleted with ID: ${id}`)
     })
 }
 
+const getContacts = (request, response) => {
+    pool.query('SELECT * FROM contacts ORDER BY created_at DESC', (error, results) => {
+        // console.log(request);
+        if (error) {
+            throw error
+        }
+        response.status(200).json(results.rows)
+    })
+}
+
+const createContacts = (request, response) => {
+    const data = request.body
+
+    pool.query('INSERT INTO contacts (application_id, user_id, full_name, position, email, phone_number, linkedin_url, notes) VALUES ($1, $2, $3, $4, $5, $6, $7, $8)', 
+    [data.application_id, data.user_id, data.full_name, data.position, data.email, data.phone_number, data.linkedin_url, data.notes], 
+    (error, results) => {
+        if (error) {
+            throw error
+        }
+        response.status(201).send(`Contact added with ID: ${results.insertId}`)
+    })
+}
+
+const updateContact = (request, response) => {
+    const id = parseInt(request.params.id)
+    const data = request.body
+
+    pool.query(
+        'UPDATE contacts SET application_id = $1, user_id = $2, full_name = $3, position = $4, email = $5, phone_number = $6, linkedin_url = $7, notes = $8 WHERE contact_id = $9',
+        [data.application_id, data.user_id, data.full_name, data.position, data.email, data.phone_number, data.linkedin_url, data.notes, id],
```

Comment:
I think we are removing the association with the application id to simplify the app